test/WebJobs.Script.Tests.Integration/Host/WebJobsScriptHostServiceTests.cs

```diff
@@ -86,6 +97,56 @@ public WebJobsScriptHostServiceTests()
             _scriptHostService = _testHost.JobHostServices.GetService<IScriptHostManager>() as WebJobsScriptHostService;
         }
 
+        [Fact]
+        public async Task GetHostKeys_DelaysUntilHostInitialized()
```

Comment:
This test is a repro for https://github.com/Azure/azure-functions-host/issues/4661